src/bundles/ipfs-provider.js

```diff
@@ -43,7 +45,27 @@ const actions = {
     const result = await getIpfs({
       providers: [
         providers.jsIpfs({
-          loadJsIpfsModule: () => require('ipfs')
+          loadJsIpfsModule: () => require('ipfs-core'),
+          options: {
+            config: {
+              Addresses: {
+                Swarm: [
+                  // '/dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+                  // '/dns4/wrtc-star2.sjc.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+                  // '/dns4/wrtc-star.discovery.libp2p.io/tcp/443/wss/p2p-webrtc-star'
+                  '/dns4/invalid-wrtc-star.discovery.libp2p.io/tcp/443/wss/p2p-webrtc-star'
+                ]
+              }
+            },
+            libp2p: ({ libp2pOptions }) => {
+              libp2pOptions.transportManager = {
+                // TODO: below seems ignored
+                // https://github.com/libp2p/js-libp2p/blob/0a6bc0d1013dfd80ab600e8f74c1544b433ece29/doc/CONFIGURATION.md#configuring-transport-manager
+                faultTolerance: FaultTolerance.NO_FATAL
+              }
+              return new Libp2p(libp2pOptions)
+            }
```

Comment:
@achingbrain was there a recent change in js-ipfs that made this type of libp2p customization not possible? It worked fine some time last year when I used this pattern for tweaking transports when running in Brave with chrome.sockets. 

Here I've explicitly set `invalid-wrtc-star` as the only one, so it always fails, and `FaultTolerance.NO_FATAL` seems to be ignored.